src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -42,9 +41,39 @@ public function getAccount() {
    * Set the current object account.
    *
    * @param AccountInterface $account
+   *   The user object.
+   * @return $this
    */
-  public function setAccount(AccountInterface $account) {
+  public function setAccount(AccountInterface $account = NULL) {
     $this->currentUser = $account;
+    return $this;
+  }
+
+  /**
+   * Get the field configuration.
+   *
+   * @param $key
+   *   Specific key from the configuration. Optional.
+   *
+   * @return array|string
+   *   Value of a configuration or all the configurations.
+   */
+  public function getConfiguration($key = NULL) {
+
+    if (!$key) {
```

Comment:
Don't think we need this. By default the key is NULL and if no value was passed `!$key` will return `TRUE` and the method will return all the configurations.